helpdesk-message-proxy-service/src/main/java/org/symphonyoss/symphony/bots/helpdesk/messageproxy/MessageProxyService.java

```diff
@@ -14,28 +15,20 @@
 import org.symphonyoss.symphony.bots.helpdesk.messageproxy.config.HelpDeskBotInfo;
 import org.symphonyoss.symphony.bots.helpdesk.messageproxy.config.HelpDeskServiceInfo;
 import org.symphonyoss.symphony.bots.helpdesk.messageproxy.config.IdleTicketConfig;
-import org.symphonyoss.symphony.bots.helpdesk.messageproxy.message.IdleMessageBuilder;
 import org.symphonyoss.symphony.bots.helpdesk.messageproxy.model.MessageProxy;
 import org.symphonyoss.symphony.bots.helpdesk.messageproxy.service.TicketService;
 import org.symphonyoss.symphony.bots.helpdesk.service.membership.client.MembershipClient;
 import org.symphonyoss.symphony.bots.helpdesk.service.model.Membership;
 import org.symphonyoss.symphony.bots.helpdesk.service.model.Ticket;
 import org.symphonyoss.symphony.clients.model.SymMessage;
 
-import java.util.HashMap;
-import java.util.Map;
-
 /**
  * Created by nick.tarsillo on 9/26/17.
  * The message proxy service handles the proxying of messages between clients and agents.
  */
 @Service
 public class MessageProxyService {
```

Comment:
You have changed the workflow for this class. Please make sure the unit tests still covering all flows.

Reply:
ok